docs/types/objecttypes.rst

```diff
@@ -62,25 +63,24 @@ passed to the ``ObjectType``.
 
 .. code:: python
 
-    import graphene
+    from graphene import ObjectType, Field, String
+
+
+    class Person(ObjectType):
+        first_name = String()
+        last_name = String()
 
-    class Person(graphene.ObjectType):
-        first_name = graphene.String()
-        last_name = graphene.String()
 
-    class Query(graphene.ObjectType):
-        me = graphene.Field(Person)
-        best_friend = graphene.Field(Person)
+    class Query(ObjectType):
+        me = Field(Person)
+        best_friend = Field(Person)
 
         def resolve_me(_, info):
```

Comment:
```suggestion
        def resolve_me(root, info):
```

Reply:
Not sure why some of these suggestions have removed a leading space, that wasn't what I was suggesting...

Reply:
Ha. No worries. Even if I missed that, I'm trying to blacken the example code, so that would be fixed anyhow, I think.